extensions/npm/main.ts

```diff
@@ -105,13 +105,13 @@ try {
 console.log(`[${green("phylum")}] Installing without build scripts…`);
 
 // Install packages without executing build scripts.
-const cmd = Deno.run({
-  cmd: ["npm", ...Deno.args, "--ignore-scripts"],
+const cmd = new Deno.Command("npm", {
+  args: [...Deno.args, "--ignore-scripts"],
```

Comment:
This is the only change in this PR where `...Deno.args` didn't become `Deno.args`. Is that intentional?

Reply:
Yes. It's intentional. In other places, removing the command name left us with just `[...Deno.args]` which is the same (value) as `Deno.args`. So I changed it to use `Deno.args` directly and avoid the copy.

Here the copy is unavoidable (since `Deno.args` is immutable and we need to add `--ignore-scripts`)